api-service/controller/oracle_databases_api.go

```diff
@@ -333,9 +333,9 @@ func (ctrl *APIController) SearchOracleDatabasesJSON(w http.ResponseWriter, r *h
 	}
 
 	if filter.PageNumber == -1 || filter.PageSize == -1 {
-		utils.WriteJSONResponse(w, http.StatusOK, databases)
+		utils.WriteJSONResponse(w, http.StatusOK, databases.Content)
 	} else {
-		utils.WriteJSONResponse(w, http.StatusOK, databases[0])
+		utils.WriteJSONResponse(w, http.StatusOK, databases.Content[0])
```

Comment:
Why are you returning just the first item in the content slice? :)
Take a look at: https://github.com/ercole-io/ercole/blob/731accfbb588349087bb716116ceb4825648e841/api-service/controller/oracle_exadata_api.go#L93

Thanks, I swear this will be the last change request ;)